Bots/DotNet/Consumers/CodeFirst/WaterfallHostBot/appsettings.json

```diff
@@ -62,6 +62,12 @@
       "AppId": "",
       "SkillEndpoint": "http://localhost:37420/api/messages"
     },
+    {
+      "Id": "ComposerSkillBotDotNet",
+      "Group": "Waterfall",
+      "AppId": "",
+      "SkillEndpoint": "http://localhost:35440/api/messages"
+    },
```

Comment:
This is not necessary, it's being addressed in this PR
https://github.com/southworks/BotFramework-FunctionalTests/pull/96/files#diff-24b95fd10b902a0b8bcb66524bd0f77f4118ee556392ed6a723e9d9bd250caf3R65-R70